index.bs

```diff
@@ -2755,14 +2820,52 @@ throws>SetUpReadableByteStreamControllerFromUnderlyingSource ( <var>stream</var>
       await writer.ready;
 
       const bytes = new Uint8Array(writer.desiredSize);
-      window.crypto.getRandomValues(bytes);
+      crypto.getRandomValues(bytes);
 
-      await writer.write(bytes);
+      // Purposefully don't await; awaiting writer.ready is enough.
+      writer.write(bytes).catch(() => {});
     }
   }
 
   writeRandomBytesForever(myWritableStream).catch(e => console.error("Something broke", e));
   </code></pre>
+
+  Note how we don't <code>await</code> the promise returned by {{WritableStreamDefaultWriter/write()}}; this would be
+  redundant with <code>await</code>ing the {{WritableStreamDefaultWriter/ready}} promise. Additionally, similar to <a
+  href="#example-manual-write-batch">a previous example</a>, we use the <code>.catch(() => {})</code> pattern on the
+  promises returned by {{WritableStreamDefaultWriter/write()}}; in this case we'll be notified about any failures
+  <code>await</code>ing the {{WritableStreamDefaultWriter/ready}} promise.
+</div>
+
+<div class="example" id="example-manual-write-dont-await">
+  To further emphasize how it's a bad idea to <code>await</code> the promise returned by
+  {{WritableStreamDefaultWriter/write()}}, consider a modification of the above example, where we continue to use the
+  {{WritableStreamDefaultWriter}} interface directly, but we don't control how many bytes we have to write at a given
+  time. In that case, the <a>backpressure</a>-respecting code looks the same:
+
+  <pre><code class="lang-javascript">
+  async function writeSuppliedBytesForever(writableStream, getBytes) {
+    const writer = writableStream.getWriter();
+
+    while (true) {
+      await writer.ready;
+
+      const bytes = getBytes();
+      writer.write(bytes).catch(() => {});
+    }
+  }
+  </code></pre>
+
+  Unlike the previous example, where—because we were always writing exactly
+  {{WritableStreamDefaultWriter/desiredSize|writer.desiredSize}} bytes each time—the
+  {{WritableStreamDefaultWriter/write()}} and {{WritableStreamDefaultWriter/ready}} promises were synchronized, in this
```

Comment:
This example (and the one above) assumes that the underlying sink returns such a promise to the writer. It might be worth adding that assumption.

Reply:
I think the advice holds even in the absence of that assumption, so I'll leave it out.